### DISCUSSION TITLE
SimpleToDoList Project RemoveItemCommand Binding Another Way

### DISCUSSION THREAD

**Discussion by artwe** [AvaloniaUI/Avalonia.Samples#101] [Q&A]
the project use this way
<Button Command="{Binding #Root.((vm:MainWindowViewModel)DataContext).RemoveItemCommand}"
Is there another way? 
I use this, but it does not work.😅 Can anyone tell me why?
<Button Command="{Binding Path=RemoveItemCommand, RelativeSource={RelativeSource AncestorType=vm:MainWindowViewModel}}"

**Comment by timunie** [comment]
Anchestor is always a control and cannot be a ViewModel, so this is why your proposal will not work.
